quipucords/api/source/serializer.py

```diff
@@ -50,6 +50,11 @@ def display_value(self, instance):
 class SourceSerializerBase(NotEmptySerializer):
```

Comment:
I'd argue `NotEmptySerializer` adds a nasty behavior we might not want on API v2. Do we need automatic removal of null fields? Lets keep this behavior on v1 for sure, for compatibility, but I'd get rid of it on v2 and embrace DRF standard.

Suggestion: use `NotEmptySerializerMixin` on SourceSerializerV1.